packages/themes/src/tokens/index.js

```diff
@@ -11,5 +11,50 @@ import { TokenGroup } from './TokenGroup';
 import { TokenSet } from './TokenSet';
 import { group } from './v11TokenGroup';
 import { set } from './v11TokenSet';
+import * as components from './components';
+import { type } from './type';
+import { layout } from './layout';
+import { v10 } from './v10';
 
 export { Token, TokenFormat, TokenGroup, TokenSet, group, set };
+
+const v11 = [
```

Comment:
@lee-chase in this case it will be the theme tokens versus anything from the IDL colors. I think colors can be pulled from `@carbon/colors` with the `colors` export IIRC (let me know if I can help out at all with that!)

Reply:
Happy to take from @carbon/colors which is where I currently retrieve the palette from. Otherwise looks good.